src/app/core/auth/auth-guard.service.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { TestBed } from '@angular/core/testing';
+import { AppState } from '@app/core';
+import { Store, StoreModule } from '@ngrx/store';
+import { MockStore, provideMockStore } from '../../../testing/utils';
```

Comment:
This import can be changed from '../../../testing/utils' to '@testing/utils' , only for cleaner code :)

Reply:
Seems like my VSCode did not catch that config 😊
I will update the PR right away